openmetadata-ui/src/main/resources/ui/src/components/common/OwnerLabel/OwnerLabel.component.tsx

```diff
@@ -57,121 +59,256 @@ export const OwnerLabel = ({
     team: boolean;
   };
   tooltipText?: string;
+  isCompactView?: boolean;
 }) => {
   const { t } = useTranslation();
   const [showAllOwners, setShowAllOwners] = useState(false);
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
```

Comment:
optimise this component.
This has become too huge and complex, split it and move parts of it to a new component or utils